### PR TITLE
add a list of dataset libraries

### PR DESCRIPTION
**Pull request by severo**
see
https://github.com/huggingface/moon-landing/pull/9352#discussion_r1627957713

### CODE REVIEW

File: packages/tasks/src/dataset-libraries.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	 */
+	repoUrl:     string;
+	/**
+	 * URL to library's docs
+	 */
+	docsUrl?:    string;
+	/**
+	 * URL to library's docs in the Hugging Face documentation
+	 */
+	hfDocsUrl?:  string;
```
Comment (julien-c) [inline packages/tasks/src/dataset-libraries.ts:26]: imo `docsUrl` can always point to hub docs, which in turn point to the library docs (like for models)

Comment (julien-c) [reply packages/tasks/src/dataset-libraries.ts:26]: that way we have one less URL to define

**Review by julien-c — APPROVED** [review]
lgtm with my suggestion

File: packages/tasks/src/dataset-libraries.ts (hunk 1 above)
Comment (severo) [reply packages/tasks/src/dataset-libraries.ts]: done

**Pull request merged**